modules/gapi/src/executor/gexecutor.cpp

```diff
@@ -202,14 +286,20 @@ void cv::gimpl::GExecutor::run(cv::gimpl::GRuntimeArgs &&args)
                 auto& out_mat = *get<cv::Mat*>(args.outObjs.at(index));
                 createMat(desc, out_mat);
             }
-            // In the case of own::Mat never reallocated, checked to perfectly fit required meta
+            // In the case of own::RMat check to fit required meta
             else
             {
-                check_own_mat();
+                check_rmat();
             }
 #else
             // Building standalone - output buffer should always exist,
             // and _exact_ match our inferred metadata
+            auto check_own_mat = [&desc, &args, &index]()
+            {
+                auto& out_mat = *get<cv::Mat*>(args.outObjs.at(index));
+                GAPI_Assert(out_mat.data != nullptr &&
+                        desc.canDescribe(out_mat));
+            };
             check_own_mat();
```

Comment:
Should we handle `RMat` as output argument in standalone mode here as well ?

Reply:
That's a good catch, thanks!

Reply:
Fixed